### ISSUE TITLE
Emby app issues

### ISSUE THREAD

**Issue by tldspectre** [spocky/miproja1#22]
The Emby app itself seems to work fine - after initial setup, it plays my movies and shows from with the app with no issues. If I attempt to play a movie or show that's listed on the Projectivy home screen, I'm always asked to enter my Emby Connect password. After doing so, Emby starts and plays the video as expected. It would be nice if typing in the password each time wasn't necessary, since it isn't when starting the app itself.

**Comment by spocky** [comment]
Does it act the same with the stock launcher ?
When an app (here: Emby) adds a program (movie, show ...) to Android's channels/programs list, it gives a few parameters such as : program name, description, thumbnail and an "intent" (see it as a link to launch this program).
When you select a program, Projectivy is just executing this intent. I don't know why this would result in requesting a password on Projectivy and not on the stock launcher.

**Comment by tldspectre** [comment]
> Does it act the same with the stock launcher ? When an app (here: Emby) adds a program (movie, show ...) to Android's channels/programs list, it gives a few parameters such as : program name, description, thumbnail and an "intent" (see it as a link to launch this program). When you select a program, Projectivy is just executing this intent. I don't know why this would result in requesting a password on Projectivy and not on the stock launcher.

I don't recall the stock launcher having the option to list individual shows/movies from Emby the way Projectivy does. I'll take a look at it to see if it does, give them a try (assuming there are any) and get back to you on the success/failure of said attempt! ;)

**Comment by tldspectre** [comment]
OK, I checked the stock launcher on my onnTV device (essentially a Chromecast with Google TV if you aren't familiar). Couldn't find any recommendations that were sourced back to Emby. I checked the Services option under settings (where you're supposedly able to add apps to the recommendations list) and Emby doesn't even show up as a choice. I also checked and verified that Emby is listed as searchable using voice search, but I get no results that include Emby as a source.

One of first things that surprised me about Projectivy when I first tried it was that it was including shows and movies available in Emby in the recommended list, which the default launcher didn't do. Unfortunately I was let down due to the password requirement issue that occurs for anything Emby-related in the recommendations list. Hence my inquiry regarding the password issue. I may do some checking on the Emby support forums to see if anyone has run into a similar situation and found a solution.

Anyway, enjoying the launcher in spite of the current limitation with Emby. BTW no issues so far with other subscription-based apps (knock on wood).
